### PR TITLE
Cut 0.2.0 release

### LINKED ISSUE #393: Complete bpfd Release 0.2
Complete bpfd Release 0.2

This Issue serves to track any issues/fixes we wan't to get in before actually releasing a 0.2.0 along with the task of actually cutting the release.

Please comment if you think we're missing any blockers 
Task
- [ ] Cut the Release
- [x] Deployment verification
- [x] #383 
- [x] #398
- [x] #419
- [x] Release note automation

### PR DESCRIPTION
**Pull request by astoycos**
This adds the changelog for bpfd's 0.2.0 release.

Updates the bpfd-operator bundle version.

Updates the bpfd, bpfctl, and bpfd-api versions to 0.2.0.

Fixes a small bug in one of our boilerplates

Fixes #393

### CODE REVIEW

File: bpfctl/Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "bpfctl"
-version = "0.1.0"
+version = "0.2.0"
 edition = "2021"
```
[the remark this answers is not in the document]
Comment (anfredette) [reply bpfctl/Cargo.toml:4]: Actually, I just looked [here](https://doc.rust-lang.org/cargo/reference/manifest.html#the-edition-field) and saw that this is the Rust Edition, and `cargo new` currently defaults it to 2021.  So, I guess this is fine.

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+## Table of Contents
+
+- [v0.1.0](#v010)
+- [v0.2.0](#v020)
+
+# v0.2.0
+
+The v0.2.0 release is our first official minor release of the bpfd project. It includes:
+
```
Comment (anfredette) [inline CHANGELOG.md:11]: Do we want to mention that we've added support for TC and Tracepoint programs?  It's implied by the CRDs below but might be better to be explicit.

**Review by anfredette — COMMENTED** [review]
Otherwise, this looks good to me.

File: CHANGELOG.md (hunk 2 above)
Comment (anfredette) [reply CHANGELOG.md:11]: Actually, should we have a section with the major new enhancements like:
- Support for TC and Tracepoint programs
- Kubernetes Operator and CRDs
- Improved documentation
- e2e testing and expanded integration tests
- Many bug fixes
- Others?

And then state what the release includes.

Comment (astoycos) [reply CHANGELOG.md]: Done! I just really included major new enhancements... Next release we'll actually be able to have a relevant change-log that's not the entire release history :)

**Review by Billy99 — APPROVED** [review]
LGTM

**Review by anfredette — APPROVED** [review]
LGTM

**Pull request merged**